modules/elt/pages/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage.adoc

```diff
@@ -99,7 +99,13 @@ For more details, please refer link:https://docs.airbyte.com/integrations/source
 === Setting the Destination Connection
 * Assuming you want to create a fresh new connection with `Teradata Vantage`, Select `Teradata Vantage` as the destination type under the "Set up the destination" section.
 * Add the `Host`, `User`, and `Password`. These are the same as the `Host`, `Username`, and `Password` respectively, used by your Clearscape Analytics Environment.
-* The default schema name is optional, however, you can provide a more appropriate name to your specific context. Here we have provided `gsheet_airbyte_td`.
+* The `Default Schema` name is optional, however, you can provide a more appropriate name to your specific context. Here we have provided `gsheet_airbyte_td`.
+
+[NOTE]
+====
+If you do not provide a `Default Schema`, you will get an error stating "Connector failed while creating schema". Make sure you provide appropriate name in the `Default Schema`.
```

Comment:
In Airbyte, although it mentions that `Default Schema` field is optional and when I keep it blank it throws an error, which should not be the case considering it is optional.  For this reason, I have added a note.

Reply:
I will edit accordingly.